docs/config.md

```diff
@@ -116,6 +116,13 @@ For a good example, see https://develop.element.io/config.json.
    1. `logo`: An HTTP URL to the avatar for the desktop build. Should be 24x24, ideally
       an SVG.
    1. `url`: An HTTP URL for where to send the user to download the desktop build.
+1. `mobileBuilds`: Used to alter promotional links to the mobile app. By default the
```

Comment:
Looking at other entries in `docs/config.md` (like `piwik`, `embeddedPages`, ...) it looks like the conscencus to describe object is as follow

```
1. `mobilesBuilds`: description...
    1. `ios`: description...
    1. `android`: description...
    1. `fdroid`: description...
```